src/app/accounting/provisioning-entries/provisioning-entries.component.html

```diff
@@ -63,7 +63,7 @@
       </ng-container>
 
       <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-      <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
+      <tr mat-row *matRowDef="let row; columns: displayedColumns;" class="select-row"></tr>
```

Comment:
Please try to implement the feature for tables which have routes defined with the rows. Else the pointer will create confusion for the user.